src/sgame/sg_bot_util.cpp

```diff
@@ -1198,15 +1198,20 @@ void BotGetIdealAimLocation( gentity_t *self, botTarget_t target, vec3_t aimLoca
 {
 	//get the position of the target
 	BotGetTargetPos( target, aimLocation );
+	bool target_buildable = BotGetTargetType( target ) == entityType_t::ET_BUILDABLE;
+	//this retrieves the target's species, to aim at weak point:
+	// * for humans, it's the head (but code only applies an offset here, with the hope it's the head)
+	// * for aliens, there is no weak point, and human bots try to take missile's speed into consideration (for luci)
+	team_t target_team = BotGetTargetTeam( target );
 
-	if ( BotGetTargetType( target ) != entityType_t::ET_BUILDABLE && BotTargetIsEntity( target ) && BotGetTargetTeam( target ) == TEAM_HUMANS )
+	if ( target_buildable == false && BotTargetIsEntity( target ) && target_team == TEAM_HUMANS )
```

Comment:
```suggestion
	if ( !target_buildable && BotTargetIsEntity( target ) && target_team == TEAM_HUMANS )
```

Reply:
dunno if the '!' is preferred over "==false". I personally prefer the '!' variant, but have to admit that it's a bit less readable. @slipher : what is unvanquished team's opinion on that?